defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-php_igbinary_version: '2.0.5'
-php_memcached_version: '3.0.4'
+php_igbinary_version: '3.0.1'
```

Comment:
Update the versions in [README.md](https://github.com/juwai/ansible-role-php-memcached/blob/master/README.md#php_igbinary_version).

Reply:
Updated